src/kubernetes/server/server-deployment.yaml

```diff
@@ -54,6 +54,8 @@ spec:
           value: 'true'
         - name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
           value: 'true'
+        - name: SPRING_CLOUD_DATAFLOW_TASK_COMPOSED_TASK_RUNNER_URI
+          value: docker://springcloud/spring-cloud-dataflow-composed-task-runner:2.5.0.BUILD-SNAPSHOT
```

Comment:
@cppwfs I see you also removed the quotes surrounded in the `value`. Is that intentional? Did you try running it on K8s with this change?